src/psyclone/tests/conftest.py

```diff
@@ -143,18 +142,36 @@ def infra_compile(tmpdir_factory, request):
     GOcean1p0Build(tmpdir)
 
 
-@pytest.fixture(scope="session")
-def parser():
+@pytest.fixture(name="_session_parser", scope="session")
+def _session_parser():
     '''
     Creates and returns an fparser object. Since this is expensive we only
-    do this once per test session (scope="session" above).
+    do this once per test session (scope="session" above). This fixture is
+    only intended to be used in the 'public' fixture `parser` below.
+
+    # TODO #1188 - move this to tests/psyir/frontend/conftest.py
```

Comment:
I would suggest either only "TODO" here without "#" or adding "#" to "TODO" in the line 164. Also, a full stop at the end for consistency with the other TODO below?

Reply:
Done.